Exercise/compute_lowest_terms.py

```diff
@@ -1,40 +1,73 @@
 # TODO: complete the function "lowest_terms" below
 
 def lowest_terms(x):
+	"""
```

Comment:
Note that while this is formatted correctly as a docstring, it does not describe what the fucntion does nor its input/outputs.

See Slack message for help on docstrings

Reply:
Effected